### ISSUE TITLE
Allrad limitiert Geschwindigkeit auf 25km/h

### ISSUE THREAD

**Issue by Frank1974** [Mogli12/GearboxAddon#390]
Hallo Stefan,
mit dem Build 67 kann man sobald das Allrad oder Differentiale eingeschalten sind nicht mehr schneller wie 25km/h fahren. Selbst wenn man vollgas gibt hört jedes Fahrzeug bei 25km/h auf zu beschleunigen.

Viele Grße -Frank.

**Comment by Mogli12** [comment]
Jein. Traktoren mit AutoStartStop schalten Allrad einfach selber aus. Alte Traktoren fahren nicht schneller. Technischer Hintergrund ist, dass zur Simulation der Voreilung der Vorderachse die Gesamtübersetzung etwas höher ist, sobald Allrad an ist.

**Comment by Mogli12** [comment]
Ich glaube, dass man bei alten Traktoren mit Allrad auf der Straße ohnehin Getreibeprobleme bekommen kann.

**Comment by Frank1974** [comment]
Nein, das Allrad kann man immer anhaben, Getriebeprobleme wird man davon nie gekommen, nur nutzten sich die Vorderreifen schneller ab.

**Comment by Mogli12** [comment]
[Allrad auf Straße - DX-Serie - Deutz Forum](http://www.deutzforum.de/index.php?thread/32017-allrad-auf-stra%C3%9Fe/)

**Comment by Frank1974** [comment]
Hallo Stefan,
wie wäre es wenn man dies einfach in das Ingame GUI oder die globals aufnimmt? dann könnte jeder für sich selbst entscheiden wie er es am realistischsten findet.

Viele Grüße -Frank

**Comment by DarcReaver** [comment]
Also Fakten oder alternative Fakten?

Ich finde das schon gut so, Allrad gehört auf den Acker, und mit 40 oder 50 km/h und Allrad auf der Straße rumfahren ist quatsch.

Eine Anmerkung hab ich auch noch dazu: mit aktivem Allrad ist der Kraftstoffbedarf höher.

**Comment by Frank1974** [comment]
Natürlich macht es keinen sinn mit Allrad auf der Straße zu fahren, aber nur weil man das Allrad an hat fährt ein Traktor plötzlich nur noch 25km/h, und darum geht es mir.

**Comment by Mogli12** [comment]
Es ist im Spiel an anderen Stellen doch auch nicht anders. Da es keinen Sinn macht, mit einem Gerät schneller als Geschwindigkeit x zu fahren, wird halt auf Geschwindigkeit x begrenzt.
